### ISSUE TITLE
Allow safe update

### ISSUE THREAD

**Issue by fubar-coder** [NuGet/Home#1635]
Enable a "Safe" update from the UI. This would prevent us from seeing updates from one major version to another - and therefore would avoid confusion.

EDIT: This would especially handy with the reintroduction of "Update All"

**Comment by csharpfritz** [comment]
What do you mean by "Safe"?  Do you want to prevent updates that cross minor version numbers, perhaps crossing major version numbers?

**Comment by fubar-coder** [comment]
Yes, that's it. It's the equivalent of the "-Safe" option of NuGet.exe

**Comment by yishaigalatzer** [comment]
@fubar-coder thanks, I was going to file this in detail today.

The design here is to have an option - Where Update All, doesn't take you above major versions for top level packages, we already have controls like that for dependencies.

This also means we don't show updates in the UI for such packages.

@fubar-coder you should know that you can limit the version range today in packages.config (although we are currently fixing a bug where it is not respected). This feature is about making a global option and not having to explicitly define it in packages.config
